packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VideoController/VideoController.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, Button, Select, ThemeContext } from 'grommet'
+import { CirclePlay, PauseFill } from 'grommet-icons'
+import counterpart from 'counterpart'
+import en from './locales/en.json'
+import FormattedTime from './FormattedTime'
+import Slider from './Slider'
+
+counterpart.registerTranslations('en', en)
+
+const customSelectTheme = {
+  textInput: {
+    extend: 'text-align: center; padding: 0; width: 40px;'
+  },
+  select: {
+    icons: {
+      margin: 'xxsmall'
+    }
+  }
+}
+
+const VideoController = ({
+  isPlaying,
+  onPlayPause,
+  onSpeedChange,
+  played,
+  playbackRate,
+  duration,
+  onSliderMouseUp,
+  onSliderMouseDown,
+  onSliderChange
+}) => {
+  let playPauseLabel = isPlaying
+    ? 'VideoController.pause'
+    : 'VideoController.play'
+
+  return (
+    <Box
+      background={{
+        dark: 'dark-3',
+        light: 'light-3'
+      }}
+    >
+      <Box pad={{ horizontal: 'xsmall', top: 'xsmall' }}>
+        <Slider
+          played={played}
+          onMouseUp={onSliderMouseUp}
+          onMouseDown={onSliderMouseDown}
+          onChange={onSliderChange}
+        />
+      </Box>
+
+      <Box direction='row' justify='between'>
+        <Box direction='row'>
+          <Box alignSelf='center' pad={{ horizontal: 'small' }}>
+            <Button
+              a11yTitle={counterpart(playPauseLabel)}
+              onClick={onPlayPause}
+              icon={isPlaying ? <PauseFill /> : <CirclePlay />}
+              plain
+            ></Button>
+          </Box>
+
+          <Box direction='row' alignSelf='center'>
+            <FormattedTime seconds={played * duration} />
+            {' / '}
+            <FormattedTime seconds={duration} />
+          </Box>
+        </Box>
+
+        <Box direction='row' alignSelf='center' pad={{ right: 'small' }}>
+          <ThemeContext.Extend value={customSelectTheme}>
+            <Select
+              a11yTitle={counterpart('VideoController.playbackSpeed')}
```

Comment:
VoiceOver/Safari is announcing this menu as `Open Drop button` but I think that's a Grommet bug in the Select component. It completely ignores your label. 😞